<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FitLog</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="Tab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>451</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Prompt</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="signin">
     <attribute name="title">
      <string>Sign in</string>
     </attribute>
     <widget class="QPushButton" name="signin_button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign in</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_signin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Prompt</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_signin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="signin_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="singup">
     <attribute name="title">
      <string>Sign up</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_signup">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_signup">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password Confirm :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_signup_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="signup_button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign up</string>
      </property>
     </widget>
     <widget class="QLabel" name="signup_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>[+] status</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Prompt</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome to FitLog</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate_New_Usert">
   <property name="text">
    <string>User</string>
   </property>
  </action>
  <action name="actionFood">
   <property name="text">
    <string>Food</string>
   </property>
  </action>
  <action name="actionCardio">
   <property name="text">
    <string>Cardio</string>
   </property>
  </action>
  <action name="actionWeight_Training">
   <property name="text">
    <string>Weight Training</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
